diesel_tests/tests/only.rs

```diff
@@ -0,0 +1,236 @@
+use crate::schema::connection;
+// use crate::schema_dsl::*;
+use diesel::dsl::*;
+use diesel::pg::expression::dsl::OnlyDsl;
+use diesel::*;
+
```

Comment:
Can we add a short comment here why we need different table definition than the default ones?

(I know that's because of inheritance, but I stop every time here and think: Why are they here?)